app/js/components/orderEntry/styles.scss

```diff
@@ -0,0 +1,24 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+.order-entry-page {
+  .error-notice{
+    color: red;
+    padding: 0px 8px;
+  }
+  
+  .order-type-button {
+    margin: 10px 5px 10px 0px;
+    outline: none;
+  }
+  
+  .order-type-button.active {
```

Comment:
there is another active class here app/js/components/labOrderEntry/styles.scss, do you think we could somehow combine both ? cos they have the same styles

Reply:
Combine the two styles would result in the `active` class to become global, I don't think this would be very nice since it is only used by the two components refrenced and considering the fact that the style itself is very little(just one line), I decided to duplicate it in the two style files.